### PR TITLE
Decrease duplicated code

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo-API/src/main/java/org/kitodo/api/docket/DocketData.java [hunk 1]
```diff
@@ -173,6 +174,9 @@ public void setTemplateProperties(ArrayList<Property> templateProperties) {
      * @return The workpieceProperties.
      */
     public ArrayList<Property> getWorkpieceProperties() {
+        if (Objects.isNull(workpieceProperties)) {
+            return new ArrayList<>();
```
Comment (henning-gerhardt) [inline Kitodo-API/src/main/java/org/kitodo/api/docket/DocketData.java]: Why did you not assign this empty list and return it?

File: Kitodo-API/src/main/java/org/kitodo/api/docket/DocketData.java [hunk 2]
```diff
@@ -192,6 +196,9 @@ public void setWorkpieceProperties(ArrayList<Property> workpieceProperties) {
      * @return The processProperties.
      */
     public ArrayList<Property> getProcessProperties() {
+        if (Objects.isNull(processProperties)) {
+            return new ArrayList<>();
```
Comment (henning-gerhardt) [inline Kitodo-API/src/main/java/org/kitodo/api/docket/DocketData.java]: Why did you not assign this empty list and return it?

File: Kitodo/src/main/java/org/goobi/production/cli/helper/WikiFieldHelper.java [hunk 3]
```diff
@@ -27,10 +27,14 @@
     private static final String TAG_INFO = "<font color=\"#0033CC\">";
     private static final String TAG_DEBUG = "<font color=\"#CCCCCC\">";
     private static final String TAG_USER = "<font color=\"#006600\">";
-    private static final String ENDTAG = "</font>";
+    private static final String TAG_END = "</font>";
 
     private static final String BREAK = "<br/>";
 
+    private WikiFieldHelper() {
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/goobi/production/cli/helper/WikiFieldHelper.java]: Why did you add this empty method?

Comment (Beacze) [reply Kitodo/src/main/java/org/goobi/production/cli/helper/WikiFieldHelper.java]: It is private constructor to exclude possibility of class instantiating. Everything in this class is static.
